src/config.c

```diff
@@ -2887,6 +2887,7 @@ standardConfig static_configs[] = {
     createBoolConfig("replica-announced", NULL, MODIFIABLE_CONFIG, server.replica_announced, 1, NULL, NULL),
     createBoolConfig("latency-tracking", NULL, MODIFIABLE_CONFIG, server.latency_tracking_enabled, 1, NULL, NULL),
     createBoolConfig("aof-disable-auto-gc", NULL, MODIFIABLE_CONFIG, server.aof_disable_auto_gc, 0, NULL, updateAofAutoGCEnabled),
+    createBoolConfig("repl-resync-on-error", NULL, MODIFIABLE_CONFIG, server.repl_resync_on_error, 0, NULL, NULL),
```

Comment:
Leave config as is, but document in redis.conf file that it may be updated in the future to be enabled.